php/Exchange.php

```diff
@@ -74,7 +74,19 @@ class Exchange {
     public $validateClientSsl = false;
     public $curlopt_interface = null;
     public $timeout = 10000; // in milliseconds
-    public $proxy = '';
+    public $proxy = ''; // for backwards compatibility
+    public $proxyUrl = null;
```

Comment:
@ttodua getting some warnings (undeclared vars) when trying to use the proxy here 
```
✗ node run-tests bybit --php --sandbox 
Debugger attached.
Testing { exchanges: ["bybit"], symbol: "all", allKeys: { '--warnings': false, '--info': false, '--ts': false, '--js': false, '--php': true, '--python': false, '--python-async': false, '--php-async': false }, exchangeSpecificFlags: { '--sandbox': true, '--verbose': false, '--private': false, '--privateOnly': false, '--info': false }, maxConcurrency: 5 } (run-tests.js:378)
[100%] Tested bybit FAIL (testExchange @ run-tests.js:291)

FAILED bybit (PHP): (explain @ run-tests.js:308)

        string(10) "\nTESTING "
        string(3) "php"
        array(2) {
          ["exchange"]=>
          string(5) "bybit"
          ["symbol"]=>
          string(3) "all"
        }
        string(2) "\n"
        PHP Deprecated:  Creation of dynamic property ccxt\bybit::$debug is deprecated in /Users/cjg/Git/ccxt12/ccxt/php/Exchange.php on line 1189
        PHP Deprecated:  Creation of dynamic property ccxt\bybit::$httpsAgent is deprecated in /Users/cjg/Git/ccxt12/ccxt/php/Exchange.php on line 1189

        Deprecated: Creation of dynamic property ccxt\bybit::$debug is deprecated in /Users/cjg/Git/ccxt12/ccxt/php/Exchange.php on line 1189

        Deprecated: Creation of dynamic property ccxt\bybit::$httpsAgent is deprecated in /Users/cjg/Git/ccxt12/ccxt/php/Exchange.php on line 1189
        PHP Warning:  Undefined variable $proxyUrl in /Users/cjg/Git/ccxt12/ccxt/php/Exchange.php on line 1462
        PHP Warning:  Undefined variable $proxyUrlCallback in /Users/cjg/Git/ccxt12/ccxt/php/Exchange.php on line 1464
        PHP Warning:  Undefined variable $httpProxy in /Users/cjg/Git/ccxt12/ccxt/php/Exchange.php on line 1466
        PHP Warning:  Undefined variable $httpsProxy in /Users/cjg/Git/ccxt12/ccxt/php/Exchange.php on line 1469
        PHP Warning:  Undefined variable $socksProxy in /Users/cjg/Git/ccxt12/ccxt/php/Exchange.php on line 1475
        PHP Warning:  Undefined variable $proxyAgentCallback in /Users/cjg/Git/ccxt12/ccxt/php/Exchange.php on line 1478

        Warning: Undefined variable $proxyUrl in /Users/cjg/Git/ccxt12/ccxt/php/Exchange.php on line 1462

        Warning: Undefined variable $proxyUrlCallback in /Users/cjg/Git/ccxt12/ccxt/php/Exchange.php on line 1464

        Warning: Undefined variable $httpProxy in /Users/cjg/Git/ccxt12/ccxt/php/Exchange.php on line 1466

        Warning: Undefined variable $httpsProxy in /Users/cjg/Git/ccxt12/ccxt/php/Exchange.php on line 1469

        Warning: Undefined variable $socksProxy in /Users/cjg/Git/ccxt12/ccxt/php/Exchange.php on line 1475

        Warning: Undefined variable $proxyAgentCallback in /Users/cjg/Git/ccxt12/ccxt/php/Exchange.php on line 1478
        PHP Warning:  Undefined variable $proxyUrl in /Users/cjg/Git/ccxt12/ccxt/php/Exchange.php on line 1462
        PHP Warning:  Undefined variable $proxyUrlCallback in /Users/cjg/Git/ccxt12/ccxt/php/Exchange.php on line 1464
        PHP Warning:  Undefined variable $httpProxy in /Users/cjg/Git/ccxt12/ccxt/php/Exchange.php on line 1466
        PHP Warning:  Undefined variable $httpsProxy in /Users/cjg/Git/ccxt12/ccxt/php/Exchange.php on line 1469
        PHP Warning:  Undefined variable $socksProxy in /Users/cjg/Git/ccxt12/ccxt/php/Exchange.php on line 1475
        PHP Warning:  Undefined variable $proxyAgentCallback in /Users/cjg/Git/ccxt12/ccxt/php/Exchange.php on line 1478

        Warning: Undefined variable $proxyUrl in /Users/cjg/Git/ccxt12/ccxt/php/Exchange.php on line 1462

        Warning: Undefined variable $proxyUrlCallback in /Users/cjg/Git/ccxt12/ccxt/php/Exchange.php on line 1464
```